public/apps/configuration/panels/role-view/role-view.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { Fragment, useEffect, useState } from 'react';
+
+import {
+  EuiBreadcrumbs,
+  EuiButton,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiSpacer,
+  EuiTab,
+  EuiTabs,
+  EuiTitle,
+} from '@elastic/eui';
+import { AppDependencies } from '../../../types';
+
+const tabs = [
+  {
+    id: 'permissions',
+    name: 'Permissions',
+    disabled: false,
+  },
+  {
+    id: 'users',
+    name: 'Users',
+    disabled: false,
+  },
+];
+
+interface RoleViewProps extends AppDependencies {
+  roleName: string;
+}
+
+export function RoleView(props: RoleViewProps) {
+  const [selectedTabId, setSelectedTabId] = useState('permissions');
+  const [breadcrumbs, setBreadcrumbs] = useState([]);
+
+  useEffect(() => {
+    setBreadcrumbs([
+      {
+        text: 'Security',
+        href: '#',
+      },
+      {
+        text: 'Roles',
+        href: '#',
+      },
+      {
+        text: props.roleName,
+      },
+    ]);
+  }, [props.roleName]);
+
+  const onSelectedTabChanged = (id) => {
+    setSelectedTabId(id);
+  };
+
+  const renderTabs = () => {
+    return tabs.map((tab, index) => (
+      <EuiTab
+        {...(tab.href && { href: tab.href, target: '_blank' })}
+        onClick={() => onSelectedTabChanged(tab.id)}
+        isSelected={tab.id === selectedTabId}
+        disabled={tab.disabled}
+        key={index}
+      >
+        {tab.name}
+      </EuiTab>
+    ));
+  };
+
+  return (
+    <>
+      <Fragment>
```

Comment:
`<>` is already fragment, why we need double fragment here?

Reply:
Good catch! Going to remove it